modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts

```diff
@@ -38,16 +38,19 @@ const sanitize = function (editor: Editor, html: string) {
     start (name, attrs, empty) {
       blocked = true;
 
-      if (name === 'script' || name === 'noscript') {
+      if (name === 'script' || name === 'noscript' || name === 'svg') {
```

Comment:
Why this change?

Reply:
Or the change below to delete attrs?

Reply:
Is it to work around the Schema weirdness you found with videos?

Reply:
We don't normally allow svg elements in the content and the tests all asserted that, however they also all had `on...` attributes.

Reply:
As discussed on slack, we'll stick with this and I'll add a test for this specifically.